Source/DataTypes/EnumConverters.cs

```diff
@@ -126,6 +126,24 @@ public sealed class SvgGradientSpreadMethodConverter : EnumBaseConverter<SvgGrad
     public sealed class SvgTextDecorationConverter : EnumBaseConverter<SvgTextDecoration>
     {
         public SvgTextDecorationConverter() : base(CaseHandling.KebabCase) { }
+
+        public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, object value)
+        {
+            if (value is string)
+                value = ((string)value).Replace(" ", ", ");
+
+            return base.ConvertFrom(context, culture, value);
+        }
+
+        public override object ConvertTo(ITypeDescriptorContext context, CultureInfo culture, object value, Type destinationType)
+        {
+            var destination = base.ConvertTo(context, culture, value, destinationType);
+
+            if (destination is string && value is SvgTextDecoration)
+                destination = ((string)destination).Replace(",", string.Empty);
```

Comment:
Are we sure that always have an additional space here? Maybe it would be more to secure to replace it with a space?

Reply:
`Enum` separator(", ") hasn't changed since long ago, so I think it's okay.
Simple replacement will result in duplicate spaces. However, I think regular expressions are expensive.
Simple replacement will result in duplicate spaces. However, I think using RegExp is cost to performance.